src/SplitPane.js

```diff
@@ -302,7 +304,15 @@ class SplitPane extends React.Component {
       });
     }
 
-    const classes = ['SplitPane', className, split, disabledClass];
+    style = Object.assign({}, style, paneContainerStyle || {});
```

Comment:
can you move this into the same statement as line 274

```suggestion
const style = { display: 'flex', ...paneContainerStyle };
```

Reply:
Great it's accepted !

Reply:
Great it's accepted !